Doc/about.rst

```diff
@@ -7,7 +7,7 @@ These documents are generated from `reStructuredText`_ sources by `Sphinx`_, a
 document processor specifically written for the Python documentation.
 
 .. _reStructuredText: http://docutils.sourceforge.net/rst.html
-.. _Sphinx: http://sphinx-doc.org/
+.. _Sphinx: http://www.sphinx-doc.org/en/stable/
```

Comment:
This replaces language agnostic link with language specific link. If the resource has versions on other languages (or will have in future) and the redirection is based on user's preferences, this replacement is not valid.

`www.` is optional.